src/components/Counter/Counter.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+type State = {
+  count: number,
+};
+
+class Counter extends React.Component<{}, State> {
+  state = {
+    count: 0,
+  };
+
+  addOne = () => {
+    this.setState((prevState) => ({
+      count: prevState.count + 1,
+    }));
+  };
+
+  add100 = () => {
+    this.setState((prevState) => ({
+      count: prevState.count + 100,
+    }));
+  };
+
+  increase = () => {
+    if (this.state.count % 5 === 0) {
+      this.add100();
+      this.addOne();
+    }
```

Comment:
```suggestion
    if (this.state.count % 5 === 0) {
      this.add100();
    }
```
У тебя `this.addOne()` и так всегда выполняется